internal/pkg/manifest/worker_svc.go

```diff
@@ -15,13 +15,11 @@ import (
 
 const (
 	workerSvcManifestPath = "workloads/services/worker/manifest.yml"
-	standardTopicType     = "standard"
-	fifoTopicType         = "fifo"
-	defaultTopicType      = standardTopicType
 )
 
 var (
-	errUnmarshalQueueOpts = errors.New(`cannot unmarshal "queue" field into bool or map`)
+	errUnmarshalQueueOpts  = errors.New(`cannot unmarshal "queue" field into bool or map`)
+	errUnmarshalFifoConfig = errors.New("unable to unmarshal fifo field into boolean or compose-style map")
```

Comment:
```suggestion
	errUnmarshalFifoConfig = errors.New(`unable to unmarshal "fifo" field into boolean or compose-style map`)
```
I guess this was because the SNS topic isn't merged yet?